midi-to-part-mp3s

```diff
@@ -26,6 +27,34 @@ class Part:
     def mp3_filepath(self):
         return self.midi_filepath.replace(".midi", ".mp3")
 
+def check_format(file_path: str) -> str:
```

Comment:
Sorry if this is a dumb question (this is one of my first Python projects... I normally work in Ruby & JS):

I'm wondering if the `-> str` bit in the method signature has any _effect_ (e.g. in conjunction with a certain python complier) or is it just documentation? I don't need a full answer — just a weblink about the feature would be enough so I can read up on it myself.

Whatever it is, I like it.

Reply:
Relatedly, this brings up an issue of consistency. If we're going to use this approach to method signatures, it should be in _all_ the functions, something we could add in a future PR.

Reply:
It‘s a type hint and just for documentation. It has no effect on Code execution but is a major improvement when working with IDEs like VSCode. The code completion feature will use it to show documentation and code hints. It's also used by plug-ins, i. e. when generating Python docstrings.

I concur: we should add it to everything in future PRs. 

https://docs.python.org/3/library/typing.html

Reply:
I have added some additional type hints and python docstrings for better consistency